wolfcrypt/src/random.c

```diff
@@ -2140,6 +2141,41 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 
     #define USE_TEST_GENSEED
 
+#elif defined(WOLFSSL_ZEPHYR)
+
+        #include <entropy.h>
+    #ifndef _POSIX_C_SOURCE
+        #include <posix/time.h>
+    #else
+        #include <sys/time.h>
+    #endif
+
+        int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
+        {
+            int ret = 0;
+            struct device* dev;
+
+            dev = device_get_binding(CONFIG_ENTROPY_NAME);
+            if (dev != NULL) {
+                if (entropy_get_entropy(dev, output, sz) != 0)
+                    ret = READ_RAN_E;
+            }
+            else {
+                word32 now;
```

Comment:
If what cases will device_get_binding() fail?  Should we allow cycle count as a way to seed outside of testing?

Reply:
Change to use the system random which uses the device if available or something else if not.